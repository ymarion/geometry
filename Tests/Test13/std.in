# Initializing the model
C c1 0 0 10
C c2 0 0 20
C c3 0 0 30
R r1 0 0 10 10
R r2 0 0 20 20
R r3 0 0 30 30
# Clearing the model
CLEAR

