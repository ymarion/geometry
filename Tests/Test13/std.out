# Creations
OK
OK
OK
OK
OK
OK
# Clear
OK
EMPTY
